ArcTouchTMDb/ArcTouchTMDb.Core/Services/API/Response/BaseResponse.cs

```diff
@@ -0,0 +1,7 @@
+﻿using System;
+namespace ArcTouchTMDb.Core.Services.API.Response
+{
+	public class BaseResponse : IBaseResponse
```

Comment:
Is there a need for this class? Seeing as it doesn't implement anything. There are a few more empty classes.